types/react-svg-pan-zoom/index.d.ts

```diff
@@ -58,6 +59,9 @@ export type ToolbarPosition = typeof POSITION_NONE | typeof POSITION_TOP | typeo
                                         typeof POSITION_BOTTOM | typeof POSITION_LEFT;
 
 export interface OptionalProps {
+    // default tool
+    defaultTool: Tool;
```

Comment:
The type Tool includes extended 'auto' value but based on documentation 'defaultTool' doesn't include the value. You can use the type Tool without auto 
 ```suggestion
    defaultTool: Exclude<Tool, typeof TOOL_AUTO>;
```